dashboard/app/assets/stylesheets/application.scss

```diff
@@ -485,6 +485,10 @@ html, body {
   a:hover {
     text-decoration: underline;
   }
+  .powered-by-aws {
+    width: 100%;
+    margin-top: 2%;
```

Comment:
*nit* I don't think we use `%` much. I have mostly seen `px`. Why did you choose % over px?

Reply:
I can change this to px! In the past I've liked to use relative spacing vs absolute